src/app/views/invoices/invoice.html

```diff
@@ -0,0 +1,41 @@
+<h1 translate>mnoe_admin_panel.dashboard.invoice.invoice_info</h1>
+<a ui-sref="dashboard.invoices" class="text-muted">
+  <i class="fa fa-chevron-left"></i><span translate>mnoe_admin_panel.dashboard.invoice.back</span>
+</a>
+<!-- invoice summary -->
+<mno-widget icon="fa-file-text-o" heading="{{'mnoe_admin_panel.dashboard.invoice.summary.widget.list.title' | translate}}">
+  <mno-widget-header> 
+  </mno-widget-header>
+  <mno-widget-body class="large no-padding">
+    <table class="table table-striped table-responsive">
+      <thead>
+        <tr>
+          <th class="col-lg-1" translate>mnoe_admin_panel.dashboard.invoice.reference</th>
+          <th class="col-lg-2" translate>mnoe_admin_panel.dashboard.invoice.reference</th>
+          <th class="col-lg-3" translate>mnoe_admin_panel.dashboard.invoice.period</th>
+          <th class="col-lg-2" translate>mnoe_admin_panel.dashboard.invoice.expected_payment_date</th>
+          <th class="col-lg-1" translate>mnoe_admin_panel.dashboard.invoice.amount</th>
+          <th class="col-lg-2" translate>mnoe_admin_panel.dashboard.invoice.status</th>
+        </tr>
+      </thead>
+      <tbody>
+        <tr>
+          <td>{{vm.invoice.slug}}</td>
+          <td>{{vm.organization.attributes.name}}</td>
+          <td>{{vm.invoice.started_at | amCalendar}} - {{vm.invoice.ended_at | amCalendar}}</td>
+          <td ng-if="vm.isPaymentEnabled">{{vm.calculatePaymentDate(vm.invoice.ended_at) | amCalendar }}</td>
```

Comment:
You have 6 `th` and 6 or 7 `td` depending on this `ng-if` condition. There should always be the same number of `th` and `td`.

Reply:
Actually, there will be 7th and 7td when ng-if=true, and 6th and 6td when ng-if=false.
"expected payment date" column should be displayed depending on paymentEnabled condition.

Reply:
You now have  7 `th` and 6 or 7 `td` depending on this `ng-if` condition. One of you `th` needs to be conditional too...